lib/rules/eqeqeq.js

```diff
@@ -153,6 +156,17 @@ module.exports = {
 					}
 					return null;
 				},
+				suggest: [
```

Comment:
The same lint message instance should not have both autofix and suggestion(s), so this suggestion should be provided only when the autofix isn't.

Reply:
My bad! Fixed.